plugins/tiledb/io/TileDBWriter.cpp

```diff
@@ -75,6 +75,80 @@ struct TileDBWriter::Args
     bool m_append;
 };
 
+NL::json attributeDefaults =
```

Comment:
It would be helpful to reflect these defaults or the behavior of the defaults into the ``writers.tiledb`` docs so that users will be able to expect how these behave. That can be done in another PR though.